resotoworker/resotoworker/__main__.py

```diff
@@ -97,6 +98,8 @@ def main() -> None:
     plugin_loader.add_plugin_config(config)
     config.load_config()
 
+    write_files_to_home_dir(config.resotoworker.write_files_to_home_dir, write_utf8_file)
```

Comment:
Can't this be part of each individual file object? Instead of having to call the write in the worker explicitly. Whenever I instantiate one of those HomedirectoryFile objects, can't they check if they need to write the file contents?